RuleDocumentation/UseCompatibleCommands.md

```diff
@@ -75,8 +75,8 @@ An example configuration might look like:
         UseCompatibleCommmands = @{
             Enable = $true
             TargetProfiles = @(
-                'ubuntu_x64_18.04_6.1.2_x64_4.0.30319.42000_core'
-                'win-4_x64_10.0.18312.0_6.1.1_x64_4.0.30319.42000_framework'
```

Comment:
Note that these are valid profile names, but not ones we ship now

Reply:
Ok, I thought I could just create my own profile and name the file `foo` and it would to file name based matching like the old `UseCompatibleCmdlets` rule?

Reply:
If you put it in the profile directory, that should work, yes. There's a method that takes a profile and turns it into a path:

https://github.com/PowerShell/PSScriptAnalyzer/blob/4f8e34e1913c670a23ac554a877b33a7c9e8a05c/Rules/CompatibilityRules/CompatibilityRule.cs#L174-L206